provisioning/roles/install-project-dependencies/tasks/main.yml

```diff
@@ -1,2 +1,5 @@
 - name: install node modules
-  npm: path=/vagrant
\ No newline at end of file
+  npm: path=/vagrant
+
+- name: install python unittest2
+  pip: name=unittest2
```

Comment:
It can be but I think in this way it's more readable and we will have every project dependencies in one file. What do you think about it?